views/dashboard-my-team.hbs

```diff
@@ -0,0 +1,104 @@
+
+    <!-- BOOTSTRAP -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
+    <!-- GOOGLE FONTS -->
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Source+Code+Pro:wght@400;500;700&display=swap" rel="stylesheet">
+
+    <!-- CSS -->
+    <link rel='stylesheet' type='text/css' href='/css/style.css' />
+    <link rel='stylesheet' type='text/css' href='/css/dashboard-my-team.css' />
+
+</head>
+
+<body>
+    <div class='wrapper'>
+
+        <!-- NAVBAR -->
+        {{> navbar }}
+
+        <div class='logo-header'>
+            <img class='img-header' src='/img/myteam/myteam.png' />
+            <span class='logo-img-header'>
+                <img class='logo-img' src='/img/myteam/myteamlogo.png' />
+            </span>
+        </div>
+
+        <div class="background theme-yellow container text-center">
+            <div>
+                <!-- MY TEAM -->
+                <div>
+                    <h1 class="title">My Team</h1>
+                    <hr class='banner-underline'>
+
+                    <table class="table">
+
+                        <thead class="labels">
+                            <tr>
+                                <th scope="col">Name</th>
+                                <th scope="col">Email</th>
+                                <th scope="col">Major</th>
+                                <th scope="col">Year</th>
+                            </tr>
+                        </thead>
+
+                        <tbody>
+
+                            {{#if team}}
+                                <div>
+                                    {{#each team}}
+                                        <tr class='teammate'>
+                                            <td>{{this.name}}</th>
+                                            <td>{{this.email}}</td>
+                                            <td>{{this.major}}</td>
+                                            <td>{{this.year}}</td>
+                                        </tr>
+                                    {{/each}}
+                                </div>
+                            {{else}}
+                                <div class="text-box no-parts">
+                                    <h2>Looks like you're not part of a team yet!</h2>
+                                </div>
+                            {{/if}}
+
+                        </tbody>
+
+                    </table>
+
+                </div>
+            </div>
+        </div>
+
+
+        <div class='filler-thick'></div>
```

Comment:
Not sure if you intended to give this div some height or something, but right now it's just an invisible div at the bottom of the page